eZ/Publish/Core/Persistence/Cache/Tests/ContentHandlerTest.php

```diff
@@ -303,30 +305,68 @@ public function providerForCachedLoadMethodsMiss(): array
         ];
     }
 
+    /**
+     * @covers \eZ\Publish\Core\Persistence\Cache\ContentHandler::updateContent
+     */
+    public function testUpdateContent()
```

Comment:
There are plenty of missing return types in here, but increasing code quality even with such small steps is worth the effort to me:
```suggestion
    public function testUpdateContent(): void
```